spec/models/associate_consultant_spec.rb

```diff
@@ -18,6 +18,22 @@
     it { should belong_to(:user) }
   end
 
+  describe '#can_graduate?' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.